news_app/lib/widgets/drawer_item.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:news_app/core/helpers/app_assets.dart';
+import 'package:news_app/core/helpers/spacing.dart';
+
+class DrawerItem extends StatelessWidget {
+  const DrawerItem({
+    super.key,
+    required this.icon,
+    required this.title,
+    required this.onTap,
+    required this.selected,
+  });
+  final String icon;
+  final String title;
+  final bool selected;
+  final void Function()? onTap;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 2),
+      child: ListTile(
+        contentPadding: const EdgeInsets.only(top: 30),
+        leading: Row(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            selected? Image.asset(
+              AppAssets.selectedIcon,
+              height: 50,
+            ) : const SizedBox(),
+            horizontalSpace(40),
```

Comment:
Make the padding outside the clickable area, and you can better use the condition like below to render or not render a widget instead of using `SizedBox()`

```
 return Padding(
      padding: const EdgeInsetsDirectional.only(start: 2, top: 30),
      child: ListTile(
        leading: Row(
          mainAxisSize: MainAxisSize.min,
          children: [
            if (selected)
              Image.asset(
                AppAssets.selectedIcon,
                height: 50,
              ),
            horizontalSpace(10),
```